xt/perltidy.t

```diff
@@ -4,7 +4,9 @@ use strict;
 use warnings;
 
 use Test::More;
-use Test::PerlTidy;
```

Comment:
Please see Makefile.PL#78

Reply:
I am surprised it never complained about missing package until the test is run.

Reply:
Maybe you always run `cpanm --notest --with-recommends --with-suggests --with-develop --installdeps .` before ... :)